client/src/components/buttons/Button.tsx

```diff
@@ -165,6 +175,44 @@ export function ButtonWithMenuV2({
   );
 }
 
+export function SingleButtonWithMenu({
+  children,
+  className,
+  color,
+  direction,
+  disabled,
+  id,
+  preventPropagation,
+  size,
+}: Omit<ButtonWithMenuV2Props, "default">) {
+  // ! Temporary workaround to quickly implement a design solution -- to be removed ASAP #3250
+  const additionalProps = preventPropagation
+    ? { onClick: (e: React.MouseEvent) => e.stopPropagation() }
+    : {};
+  return (
+    <UncontrolledDropdown
+      {...additionalProps}
+      className={cx(className)}
```

Comment:
Do not "cx" a single classname:

```suggestion
      className={className}
```

Reply:
Fixed in f8b261e1b60971d782abd74d96fae55dba5acae5